### PR TITLE
[Communication] - Identity - Functions and objects renaming

### PR DESCRIPTION
**Pull request by jbeauregardb**

### CODE REVIEW

**Review by minnieliu — APPROVED** [review]

File: sdk/communication/azure-communication-identity/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,13 @@
 ### Added
 - Added a retryPolicy() chain method to the `CommunicationIdentityClientBuilder`
 
+### Breaking
+- `CommunicationIdentityClient.createUserWithToken` and `CommunicationIdentityAsyncClient.createUserWithToken` functions have been renamed to
```
Comment (RezaJooyandeh) [inline sdk/communication/azure-communication-identity/CHANGELOG.md:8]: let's add "." at the end of the lines.

File: sdk/communication/azure-communication-identity/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,13 @@
...
+`CommunicationIdentityClient.createUserAndToken` and `CommunicationIdentityAsyncClient.createUserAndToken`
+- `CommunicationIdentityClient.createUserWithTokenWithResponse` and `CommunicationIdentityAsyncClient.createUserWithTokenWithResponse` functions have been renamed to
```
Comment (RezaJooyandeh) [inline sdk/communication/azure-communication-identity/CHANGELOG.md:10]: do we normally say function or method in java?

File: sdk/communication/azure-communication-identity/CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,13 @@
...
+`CommunicationIdentityClient.createUserAndTokenWithResponse` and `CommunicationIdentityAsyncClient.createUserAndTokenWithResponse`
+- `CommunicationUserIdentifierWithTokenResult` object has been renamed to `CommunicationUserIdentifierAndTokenResult`
```
Comment (RezaJooyandeh) [inline sdk/communication/azure-communication-identity/CHANGELOG.md:12]: I think we should say class "instead" of "object"

File: sdk/communication/azure-communication-identity/CHANGELOG.md (hunk 2 above)
Comment (jbeauregardb) [reply sdk/communication/azure-communication-identity/CHANGELOG.md:10]: Python is starting to break me 💯

Comment (minnieliu) [reply sdk/communication/azure-communication-identity/CHANGELOG.md:10]: Hmm actually, the examples below don't specify "functions"/"methods". 

e.g.: `CommunicationIdentityAsyncClient.issueToken` and `CommunicationIdentityClient.issueToken` is renamed to `CommunicationIdentityAsyncClient.getToken` and `CommunicationIdentityClient.getToken`.

**Pull request merged**